pkg/services/datasources/service/datasource_service.go

```diff
@@ -312,6 +312,7 @@ func (s *Service) httpClientOptions(ds *models.DataSource) (*sdkhttpclient.Optio
 		Timeouts: timeouts,
 		Headers:  s.getCustomHeaders(ds.JsonData, s.DecryptedValues(ds)),
 		Labels: map[string]string{
+			"datasource_type": ds.Type,
```

Comment:
good catch! i apparently got a false positive during testing because prometheus still uses proxy mode for resource calls. i'll update this

Reply:
@wbrowne since prometheus is split between proxy calls for resources and backend calls for queries, will enabling logging for prometheus result in two instances of the logger? one for the proxy, and one for the queries?

i'm wondering how to handle that if there are two instances. it seems like currently there could be issues because they would both write to the same file, but the separate instances don't share a mutex. 🤔

Reply:
Yeah that sounds correct unfortunately. Will have a think if there's something we can do about that. Would a HAR file per client be awkward/weird to work with?

I wonder how many other plugins are still using both - not many AFAIK

Reply:
i think i have a solution here: https://github.com/grafana/grafana-plugin-sdk-go/pull/491